src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image {
+    height: 134px;
+    width: 160px;
```

Comment:
Magic numbers in the code: It's a good practice to avoid magic numbers in your code. Magic numbers are numeric values that may change in the future, and they are 'magic' because it's not clear what they're for. It's better to use SCSS variables for such values. For example, you could create a variable `$image-height: 134px;` and `$image-width: 160px;` and then use it here.